src/NServiceBus.AcceptanceTests/ApiExtension/When_extending_sendoptions.cs

```diff
@@ -5,7 +5,7 @@
     using AcceptanceTesting;
     using EndpointTemplates;
     using Extensibility;
-    using Pipeline;
+    using NServiceBus.Pipeline;
```

Comment:
What is the desired format for using statements? I prefer to be explicit like this, but I think a new setting was introduced remove this?